packages/config/prettierrc.web-sdk.cjs

```diff
@@ -0,0 +1,9 @@
+module.exports = {
```

Comment:
this file should probably be inside web-sdk, or called `prettierrc.svelte.cjs`

Reply:
Putting it inside web-sdk would be the opposite of our chat regarding moving configs to a single centralized source of truth. I considered .svelte. but I'd like the configs to be based on the package that consumes them, Svelte is the part that may change as well.